packages/sdk/src/restclient.spec.ts

```diff
@@ -581,9 +582,15 @@ describe("RestClient", () => {
       {
         const result = await executeContract(client, pen, contractAddress);
         expect(result.code).toBeFalsy();
-        // console.log("Raw log:", result.raw_log);
-        const [firstLog] = parseLogs(result.logs);
-        expect(firstLog.log).toEqual(`released funds to ${beneficiaryAddress}`);
+        // console.log("Raw log:", result.logs);
+        const logs = parseLogs(result.logs);
```

Comment:
I'd love some tests on transfer, especially when we send to the contract and contract sends to the beneficiary in the same transaction. I think it all ends up mixed up.

Something happens in the higher levels outside the module and good to check that, so I can try to adjust the server before final repease